docs/customer-360/customer-journey/process.sql

```diff
@@ -0,0 +1,24 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of pages
+CREATE STREAM pages (
+  customer INTEGER,
+  time BIGINT,
+  page_id STRING,
+  page STRING
+) WITH (
+  VALUE_FORMAT = 'JSON',
+  KAFKA_TOPIC = 'pages',
+  PARTITIONS = 6
```

Comment:
If we setup a connector, the topic should already exist and we don't need to specify the number of partitions?

(I leave the same comment on the other PR review -- not I am wondering about the case when users use the manual `INSERT INTO` path... maybe we can keep it but clarify that it only applies is the topic does not exist yet?)

Reply:
@mjsax will track separately in https://github.com/confluentinc/ksqldb-recipes/issues/156